peachjam/js/components/FindDocuments/AdvancedSearchFields.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div class="">
+    <div class="form-title">
```

Comment:
Prefer an h* tag, rather than custom styling, so it's consistent across the site. Perhaps `h4`?